google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -153,9 +148,20 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    configuration.setCloudProjectName(projectSelector.getText());
-    configuration.setCloudSdkExecutablePath(
-        CloudSdkUtil.toExecutablePath(cloudSdkDirectoryField.getText()));
+    if (StringUtil.isEmpty(cloudSdkDirectoryField.getText())
+        || !CloudSdkUtil.containsCloudSdkExecutable(cloudSdkDirectoryField.getText())) {
+      throw new RuntimeConfigurationError(
+          GctBundle.message("appengine.cloudsdk.location.missing.warning.message"));
```

Comment:
I think this is "Please select a Cloud SDK directory" which does not sound like a descriptive exception message here